libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -543,8 +543,8 @@ Vector3f AP_AHRS_NavEKF::wind_estimate(void) const
 // return an airspeed estimate if available. return true
 // if we have an estimate
 bool AP_AHRS_NavEKF::airspeed_estimate(float &airspeed_ret) const
-{
-    return AP_AHRS_DCM::airspeed_estimate(airspeed_ret);
+{    
```

Comment:
Not a blocker but a small white space issue here..

Reply:
fixed, thanks